faucet-frontend/src/main.js

```diff
@@ -145,3 +163,14 @@ document.querySelector('#add-emerald-button').addEventListener('click', () => {
         alert('The Oasis Emerald Testnet RPC already added.');
     });
 });
+
+async function checkFaucetBalance() {
+  const acc = await (await fetch('https://testnet.nexus.oasis.io/v1/consensus/accounts/oasis1qzna6nq9kuktjmxx2s84z38eysqyts84jc9lgdg2')).json()
```

Comment:
Please make the check balance API link configurable.

The Testnet Faucet address should be configurable and preferably not limited to Testnet (and Consensus). We want to deploy multiple Faucets in the future (e.g. a password-protected Testnet Faucet) which will have its own address.